xla/service/gpu/runtime/annotation.cc

```diff
@@ -42,29 +42,22 @@ limitations under the License.
 #include "tsl/profiler/lib/nvtx_utils.h"
 #include "tsl/profiler/lib/scoped_annotation.h"
 
+#if GOOGLE_CUDA
```

Comment:
The custom payload schema registration uses many NVTX-defined types and macros. At the moment, this file (`annotation.cc`) is the only file in XLA that uses those types and macros, so it seemed pragmatic to let this + `nvtx_utils.cc` be the only two places that actually include NVTX headers, rather than inventing new wrappers just for this.

Reply:
👍 ok, np, as long as it doesn't leak and spread it's fine